doc/03_devops_integration.md

```diff
@@ -0,0 +1,120 @@
+2. Log in with *LOGIN VIA OIDC PROVIDER* and use your Intershop Azure account.
+3. Select *User Profile* on top left of the page.
+4. In the dialog you can copy the CLI secret.
+
+In order to build the IOM project locally, you have to always log in to *docker.tools.intershop.com* using your Intershop Azure account and the CLI secret:
+
+    docker login docker.tools.intershop.com
+
+### Create Kubernetes secret *intershop-pull-secret*
+
+For running the IOM project in *devenv-4-iom*, an *image pull secret* (Kubernetes secret object) has to be created for *docker.tools.intershop.com*. The *image pull secret* has to be created within the default namespace of Kubernetes.
+
+    kubectl create secret docker-registry intershop-pull-secret \
+      --context="docker-desktop" \
+      --docker-server=docker.tools.intershop.com \
+      --docker-username='<your Intershop Azure account>' \
+      --docker-password='<CLI secret, see above>'
+
+### Add the pull-secret *intershop-pull-secret* to configuration of devenv-4-iom
+
+Finally, the name of the newly created Kubernetes secret has to be passed to *devenv-4-iom*. To do so, set the key *IMAGE_PULL_SECRET* within the user-specific configuration file of *devenv-4-iom* (devenv.user.properties):
+
+    # change into the root directory of IOM project
+    echo IMAGE_PULL_SECRET=intershop-pull-secret >> devenv.user.properties
+
+## Get Access to *Project Docker Repository*
+
+The *Project Docker Repository* (*Project Docker Repo* in the figure above) contains all project-specific Docker images, which have passed the CI process of the *Azure DevOps Environment*. The Docker images of this repository should be used by *devenv-4-iom* to run local customizations/configurations on top of them.
+
+Since the *Project Docker Repository* is a private Docker registry too, a second *image pull secret* has to be created. For this second *image pull secret* an access token is required. 
+
+### Login into Azure
+
+    # Use the azure-cli Docker image, to avoid local installation of Azure CLI tools.
+    # See https://docs.microsoft.com/en-us/cli/azure/run-azure-cli-docker
+    docker run -it mcr.microsoft.com/azure-cli
+
+    # Login with your Azure account and your Azure password.
+    az login --username <your Intershop Azure account> --password <your Intershop Azure password>
+
+### Get the name of the *Project Docker Repository*
+
+The *Project Docker Repository* is an *Azure Container Registry* (*ACR*). It is named after organization owning the *Azure DevOps Environment*. If you are not sure about the correct name of the ACR, you can get a list of all ACRs within your Azure Subscription. In this case, you need to know your Azure Subscription-ID.
+
+    # Execute the command within running azure-cli Docker container.
+    # Look for a name, that is matching your organization. This name has to be
+    # used in subsequent commands.
+    az acr list --subscription <your Azure Subscription-ID> --out table
```

Comment:
this is a bit complicated - the ACR name should be visible in the System Overview page provided by OPS for all customers

Reply:
It would be good to know, how a partner is really seeing his system. If it is clear how the partner will get all the information, I will change this. In the meantime, I will leave it as it is.